### PR TITLE
SW-1526 support GPS coordinates in add acession v2

### PR DESCRIPTION
**Pull request by karthikbtf**
- using npm coordinate-parser which captures our requirements for parsing
- using lodash for array deep equals
- tested and successfully saved/retrieved accession with coords
- i have a separate ticket for error validation across the create accession form

Terraware App 2022-09-07 11-25-54

Terraware App 2022-09-07 11-26-15

Terraware App 2022-09-07 11-26-30

Terraware App 2022-09-07 11-26-57

### CODE REVIEW

File: src/api/types/accessions.ts [hunk 1]
```diff
@@ -20,3 +20,7 @@ export const photoEndpoint = '/api/v1/seedbank/accessions/{id}/photos/{photoFile
 export const checkInEndpoint = '/api/v1/seedbank/accessions/{id}/checkIn';
 
 export type Accession = AccessionGetResponse['accession'];
+
+export const schemas = 'schemas';
+export type Geolocation = components[typeof schemas]['Geolocation'];
+console.log(Geolocation, 'wut');
```
Comment (constanzauanini) [inline src/api/types/accessions.ts:26]: Remove?

File: src/components/accession2/Accession2GPS.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            label={index === 0 ? strings.LATITUDE_LONGITUDE : ''}
+            type='text'
+          />
+          {index !== 0 && (
+            <IconButton
+              id={`delete-gps-coords${index}`}
+              aria-label='delete'
+              size='small'
+              onClick={() => onDeleteGpsCoords(index)}
+              className={classes.delete}
```
Comment (constanzauanini) [inline src/components/accession2/Accession2GPS.tsx:113]: Why didn't you use sx prop for this?

File: src/api/types/accessions.ts (hunk 1 above)
Comment (karthikbtf) [reply src/api/types/accessions.ts:26]: oops, thanks, will remove

File: src/components/accession2/Accession2GPS.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/accession2/Accession2GPS.tsx:113]: copy pasta from another component,, will switch to sx

**Review by constanzauanini — APPROVED** [review]
Thanks!

**Pull request merged**